crates/sui-framework/packages/sui-framework/sources/display_registry.move

```diff
@@ -0,0 +1,95 @@
+module sui::display_registry;
```

Comment:
Similar to suggestions on the CoinMetadata PR, I believe the registry type and fields structure should be simplified to:

```
public struct Registry has key {
    id: UID,
    registry: Table<TypeName, VecMap<String, String>>,
}
```